phpseclib/Crypt/Base.php

```diff
@@ -296,9 +296,9 @@ class Crypt_Base
     /**
      * Does internal cipher state need to be (re)initialized?
      *
-     * @see setKey()
-     * @see setIV()
-     * @see disableContinuousBuffer()
+     * @see Crypt_Base::setKey()
```

Comment:
meh, I don't really mind - I was just throwing the idea out there incase you liked it :)